src/chainparams.cpp

```diff
@@ -317,6 +315,7 @@ class SigNetParams : public CChainParams {
         consensus.fPowNoRetargeting = false;
         consensus.nRuleChangeActivationThreshold = 1916;
         consensus.nMinerConfirmationWindow = 2016;
+        consensus.MinBIP9WarningHeight = 0;
```

Comment:
fadb384e perhaps group `MinBIP9WarningHeight` with the other height settings as done in the other three chains
 ```diff
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -309,13 +309,13 @@ public:
         consensus.BIP66Height = 1;
         consensus.CSVHeight = 1;
         consensus.SegwitHeight = 1;
+        consensus.MinBIP9WarningHeight = 0;
         consensus.nPowTargetTimespan = 14 * 24 * 60 * 60; // two weeks
         consensus.nPowTargetSpacing = 10 * 60;
         consensus.fPowAllowMinDifficultyBlocks = false;
         consensus.fPowNoRetargeting = false;
         consensus.nRuleChangeActivationThreshold = 1916;
         consensus.nMinerConfirmationWindow = 2016;
-        consensus.MinBIP9WarningHeight = 0;
```

Reply:
I've put it in the line right before powLimit, like in all other chain constructors


In either case I might add compiler warnings to avoid uninitialized reads here in the consensus code after we switch to  C++17, and that requires rewriting everything anyway ;)